src/app/modules/category/views/performance/performance.component.ts

```diff
@@ -83,6 +78,12 @@ export class CategoryPerformanceComponent extends CategoryBase implements OnDest
   public _showExternalLink = true;
   public _showCustomLegend = false;
 
+  public _tableModesOptions = [
+    ...(this._viewConfig.userFilter !== null ? { label: this._translate.instant('app.engagement.dimensions.users'), value: TableModes.users } : []),
+    ...(this._viewConfig.entryFilter !== null ? { label: this._translate.instant('app.engagement.dimensions.entries'), value: TableModes.entries } : []),
+    { label: this._translate.instant('app.engagement.dimensions.dates'), value: TableModes.dates }
+  ];
```

Comment:
@adimachmali when we have only 1 option in the dropdown (dates) - I think we can hide the dropdown completely, both in category and entry views.

Reply:
Fixed